simplq/src/components/pages/Join/index.jsx

```diff
@@ -14,28 +15,33 @@ import InputField from '../../common/InputField';
 
 export function JoinQueueWithDetails(props) {
   const queueId = props.match.params.queueId;
```

Comment:
I'll split up these components into different files and name them better. I intended to do it later but forgot.

Reply:
No lets not. As per https://github.com/SimplQ/simplQ-frontend/issues/121 we would ditch that intermediate page...

And always when you get such ideas during development, raise an issue and move on. It's pretty effective. You get bonus points if you leave a TODO comment in the code as well with a link to the issue.

Reply:
Yeah, if that's the path we're going with, then we can ditch this altogether.